src/mod.ts

```diff
@@ -77,7 +77,8 @@ export async function createPlugin(
   opts.config ??= {};
   opts.fetch ??= fetch;
 
-  opts.runInWorker ??= CAPABILITIES.hasWorkerCapability;
+  // TODO(chrisdickinson): reset this to `CAPABILITIES.hasWorkerCapability` once we've fixed https://github.com/extism/js-sdk/issues/46.
+  opts.runInWorker ??= false;
```

Comment:
(It's a blink-and-you'll-miss-it diff – this is what _actually_ changes the default threading behavior of the plugin.)